### PR TITLE
Update harbour-vodman-sv.ts

### PR DESCRIPTION
**Pull request by eson57**
new and changed strings translated

### CODE REVIEW

File: src/vodman-app/translations/harbour-vodman-sv.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@
     <message id="about-sources-text">
         <location filename="../qml/pages/AboutPage.qml" line="149"/>
         <source>Sources are on Github</source>
-        <translation>Källkoder finns tillgängliga på &lt;a href=&quot;https://github.com/jgressmann/harbour-vodman&quot;&gt;GitHub&lt;/a&gt;.</translation>
+        <translation>Källkoder finns på GitHub</translation>
```
Comment (jgressmann) [inline src/vodman-app/translations/harbour-vodman-sv.ts:73]: Please add the link back
```suggestion
        <translation>Källkoder finns på &lt;a href=&quot;https://github.com/jgressmann/harbour-vodman&quot;&gt;GitHub&lt;/a&gt;.</translation>
```

File: src/vodman-app/translations/harbour-vodman-sv.ts [hunk 2]
```diff
@@ -340,7 +340,7 @@
         <location filename="../qml/pages/DownloadPage.qml" line="909"/>
         <source></source>
         <oldsource>Copy file path to clipboard</oldsource>
-        <translation type="unfinished">Kopiera filväg till urklipp</translation>
+        <translation type="unfinished"></translation>
```
Comment (jgressmann) [inline src/vodman-app/translations/harbour-vodman-sv.ts:343]:
```suggestion
        <translation>Kopiera filväg till urklipp</translation>
```

**Review by jgressmann — CHANGES_REQUESTED** [review]

**Comment by jgressmann** [comment]
Thank you for your contribution. I merged your changes but had to update/fix some strings (see review notes). You may want to have another look.

**Comment by eson57** [comment]
Looks OK to me.

**Pull request merged**